fem/coefficient.cpp

```diff
@@ -319,6 +321,31 @@ void MatrixFunctionCoefficient::Eval(DenseMatrix &K, ElementTransformation &T,
    }
 }
 
+void MatrixFunctionCoefficient::EvalSymmetric(Vector &K,
```

Comment:
Can you remind why you need this new method, and you can't just do a check inside `Eval`?
```c++
void MatrixFunctionCoefficient::Eval(DenseMatrix &K, ElementTransformation &T,
                                     const IntegrationPoint &ip)
{
  if(symmetric)
  {
  ...
  }
  else
  {
  ...
  }
}
```

Reply:
@YohannDudouit In the symmetric case, it returns a `Vector` of 6 values in 3D instead of a `DenseMatrix` of 9 values.